### DISCUSSION TITLE
Dodona not automatically picking up Docker image version updates

### DISCUSSION THREAD

**Discussion by m0n0chr0m3** [dodona-edu/feedback-and-support#38] [Questions - Vragen] [answered]
Is the Dodona platform supposed to automatically pick up changes made to [the docker-images repo](https://github.com/dodona-edu/docker-images), or do I need to take a specific action to trigger an update?

## In case of the latter
What do I have to do to update the image used?

## In case of the former
Something seem to have broken.

The [docker image for the Scheme judge](https://github.com/dodona-edu/docker-images/blob/master/dodona-scheme.dockerfile) builds on the Racket image for version 8.13. Yet, I'm getting multiple indications that the version used by the current Dodona platform is in fact Racket version 8.11.1:
- running the Scheme expression `(version)` inside an exercise or test yields the string `"8.11.1"`, and
- another error I'm getting complains about not finding something inside `/home/runner/.local/share/racket/8.11.1/collects`.

[Commit 8d83c15cb1e0809e2d53830e769f57d6eb5ee33d to the docker-images repo](https://github.com/dodona-edu/docker-images/commit/8d83c15cb1e0809e2d53830e769f57d6eb5ee33d) should have been bumped the Racket version to 8.12, and it has since been bumped to 8.13. But I'm not seeing the effects of that inside the Dodona platform.

It does not appear to be an issue on the side of the Racket image: if I locally build and run the dodona-scheme.dockerfile, it contains Racket 8.13 as expected.

**Comment by niknetniko** [comment, marked as answer]
This does not happen automatically; we (Team Dodona) need to manually trigger a deploy to get the latest images to the workers. I'll do this now, so the new image should be available soon (EDIT: the images are now [deployed](https://github.com/dodona-edu/dodona/actions/runs/9287110815/job/25555324729)).

You can see these deployments here: https://github.com/dodona-edu/dodona/actions/workflows/deploy-images.yml
